src/device/device-management.ts

```diff
@@ -108,7 +139,10 @@ export async function getDevices(deviceKind?: media.DeviceKind): Promise<MediaDe
       media.enumerateDevices
     );
   } catch (error) {
-    throw new WcmeError(ErrorTypes.DEVICE_PERMISSION_DENIED, 'Failed to ensure device permissions');
+    throw new WebrtcError(
```

Comment:
shouldn't we be throwing this error directly from `media.ensureDevicePermissions`?